sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -719,8 +719,9 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
 
       jsSourceDirectories ++= (jsSourceDirectories in Compile).value,
 
-      // Default to deprecated requiresDOM to not break old build.
-      requireJsDomEnv := requiresDOM.?.value.getOrElse(false),
+      // Default to deprecated scalaJSRequestsDOM to not break old builds.
+      // i.e. `requiresDOM := true` or `jsDependencies += RuntimeDOM`
+      requireJsDomEnv := scalaJSRequestsDOM.?.value.getOrElse(false),
```

Comment:
`scalaJSRequestsDOM` is always defined, so you can just use `.value` instead of `.?.value.getOrElse(false)`.

Reply:
Thanks, I've pushed fix.